weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -303,6 +308,11 @@ function buildCallsTableColumns(
   }) as GridColumnGroupingModel;
 
   for (const key of allDynamicColumnNames) {
+    // We don't want to show usage stats in the table, because we have a separate derived column for that.
+    if (key.startsWith('summary.usage')) {
```

Comment:
sorry to comment here again, but there is aline above `const tree = buildTree([...allDynamicColumnNames]);` that should filter this too.

I think you want something like:
```
const usableDyanmicColumnNames = allDynamicColumnNames.filter(...)
```
then use `usableDyanmicColumnNames` where `allDynamicColumnNames` is now

Reply:
makes sense